src/appengine/handlers/jobs.py

```diff
@@ -44,6 +57,37 @@ def get_queues():
   return queues
 
 
+def get_results(this):
+  """Get results for the jobs page."""
+  params = {k: v for k, v in this.request.iterparams()}  # pylint: disable=unnecessary-comprehension
+
+  query = datastore_query.Query(data_types.Job)
+
+  # Results sorted alphabatically by name.
+  # is_desc reverses the sorting.
+  query.order('name', is_desc=False)
+
+  page = helpers.cast(
+      this.request.get('page') or 1, int, "'page' is not an int.")
+
+  # Adding Filters for Search Query
```

Comment:
No need for this comment, as it's pretty self evident.

Reply:
sure, will remove this